scss/main.scss

```diff
@@ -1141,6 +1141,151 @@ p.cardText {
   align-items: center;
 }
 
+.content-container {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 80px 96px;
+  max-width: 1440px;
+  height: max-content;
+  .content-header {
+    h2 {
+      padding: 16px 0px 16px 0px;
+      margin-bottom: 0px;
+      margin-left: auto;
+      margin-right: auto;
+      width: 600px;
+      height: auto;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 3rem;
+      line-height: 62px;
+      color: $darker-grey;
+    }
+    p {
+      margin-bottom: 24px;
+      margin-left: auto;
+      margin-right: auto;
+      width: 600px;
+      height: auto;
+      font-family: $font-family-base;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 1.25rem;
+      line-height: 30px;
+      color: $darker-grey;
+    }
+  }
+  
+  @media (max-width: 768px) {
+    padding: 64px 48px;
```

Comment:
The horizontal margin on smaller screens should be 24px

<img width="153" alt="Screen Shot 2021-05-04 at 7 39 18 PM" src="https://user-images.githubusercontent.com/9326234/117085851-8e18bd00-ad10-11eb-9309-c5de30c73db1.png">